src/index.js

```diff
@@ -1,3 +1,4 @@
+var get = require('lodash/get')
```

Comment:
@christianalfoni don't you mind on using lodash here? we are importing only separate function.